src/odemis/driver/pigcs.py

```diff
@@ -580,7 +580,8 @@ def GetAxes(self):
         # SAI? (Get List Of Current Axis Identifiers)
         # SAI? ALL: list all axes (included disabled ones), one per line
         answer = self._sendQueryCommand("SAI? ALL\n")
-        axes = set(a for a in answer)
+        answer = list([answer]) if not isinstance(answer, list) else answer
+        axes = set([a.decode('ascii') for a in answer])
```

Comment:
Use "latin1" instead of "ascii", to be sure there is no issue with unusual characters.